shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelper.java

```diff
@@ -35,6 +35,12 @@ private HapiHelper() {}
     public static final StringType EXTENSION_DATA_TYPE_HD1 = new StringType("HD.1");
     public static final String EXTENSION_XPN7 = "XPN.7";
```

Comment:
This might not be part of the scope of work for this PR, but we could refactor these extension HL7 names above into [here](https://github.com/CDCgov/trusted-intermediary/blob/main/etor/src/main/java/gov/hhs/cdc/trustedintermediary/plugin/path/Hl7FhirMappingPath.java#L101). Thoughts?